src/audits/performance/first-meaningful-paint.js

```diff
@@ -114,13 +117,15 @@ class FirstMeaningfulPaint extends Audit {
     // * fMP webfont: basic + waiting for in-flight webfonts to paint
     // * fMP full: considering both page height + webfont heuristics
 
-    // We're interested in the last of these
-    const lastfMPts = data.fmpCandidates
-      .map(e => e.ts)
-      .reduce((mx, c) => Math.max(mx, c));
+    // Calculate the difference from navigation and save all candidates
+    let timings = {};
+    Object.keys(data.fmpCandidates).forEach(name => {
+      const evt = data.fmpCandidates[name];
+      timings[name] = evt && ((evt.ts - data.navStart.ts) / 1000);
+    });
 
-    // First meaningful paint
-    const firstMeaningfulPaint = (lastfMPts - data.navStart.ts) / 1000;
+    // First meaningful paint is the last timestamp observed from the candidates
+    const firstMeaningfulPaint = Object.values(timings).reduce((mx, c) => max(mx, c));
```

Comment:
`Object.values` -- This appears to still be behind a flag, even in node 6/v8 5.3 if I'm not missing something